tests/rptest/services/redpanda.py

```diff
@@ -1666,6 +1666,21 @@ def scale_cluster(self, nodes_count):
         """
         return self._cloud_cluster.scale_cluster(nodes_count)
 
+    def clean_cluster(self):
```

Comment:
Should this be called in stop() as we always want to call this, rather than relying on each user of the cloud tests to call it?

This will be more important I think as we try to expand cloud tests and possibly unify them with vanilla DT tests: the service should DTRT without the test class needing to know about it.

Reply:
I say that topic is a 'user level' resource and it is better to have this called from setup/teardown for visibility purposes.

Reply:
i'll keep its usage in the `tearDown()` method for now since we may be soon refactoring classes in the `redpanda.py` from our discussion at our last sync.